content/realtime/channels/overview.textile

```diff
@@ -140,9 +140,34 @@ bc[objc]. ARTRealtimeChannel *channel = [realtime.channels get:@"channelName"];
 
 bc[swift]. let channel = realtime.channels.get("channelName")
 
-h4(#setting-channel). Setting channel options and encryption
+h4(#setting-channel-options). Setting channel options
 
-A set of "channel options":#channel-options may also be passed to configure a channel for encryption. Find out more about "symmetric message encryption":/realtime/encryption.
+You can set "channel options":#channel-options in one of two ways:
```

Comment:
Can we explain to the reader why they want to set options pleawe?

Reply:
@tom any ideas on what we can generically say regarding why someone would want to set options, not referring to the specific options as they are already described.

Reply:
One for @paddybyers or @SimonWoolf I would say as this API I believe was introduced through their thinking.